app/helpers/patient_details_helper.rb

```diff
@@ -41,12 +41,12 @@ def linelist
       end_of_monitoring: (continuous_exposure ? 'Continuous Exposure' : end_of_monitoring) || '',
       risk_level: exposure_risk_assessment || '',
       monitoring_plan: monitoring_plan || '',
-      latest_report: latest_assessment_at&.rfc2822 || '',
-      transferred_at: latest_transfer_at&.rfc2822 || '',
+      latest_report: latest_assessment_at || '',
+      transferred_at: latest_transfer_at || '',
       reason_for_closure: monitoring_reason || '',
       public_health_action: public_health_action || '',
       status: status&.to_s&.humanize&.downcase&.gsub('exposure ', '')&.gsub('isolation ', '') || '',
-      closed_at: closed_at&.rfc2822 || '',
+      closed_at: closed_at || '',
       transferred_from: latest_transfer&.from_path || '',
       transferred_to: latest_transfer&.to_path || '',
       expected_purge_date: expected_purge_date,
```

Comment:
I exported the exposure line list CSV and noticed that `expected_purge_date` is in the format `Thu, 04 Feb 2021 16:07:06 +0000`. Is that change in the scope of this PR?

Reply:
I think this would be in scope, yes - good catch

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/665/commits/93b4918cc27f51537f467aede79ce6587367079a